packages/flutter_tools/lib/src/bundle.dart

```diff
@@ -48,6 +48,46 @@ const String _kVMSnapshotData = 'vm_snapshot_data';
 const String _kIsolateSnapshotData = 'isolate_snapshot_data';
 const String _kIsolateSnapshotInstr = 'isolate_snapshot_instr';
 
+class BundleFactory {
```

Comment:
Not sure I understood this part. What's the intent here?

Reply:
So, we can mock `buildBundle`. I added a comment.